backend/src/main/java/com/library/backend/service/MealService.java

```diff
@@ -0,0 +1,22 @@
+package com.library.backend.service;
+import com.library.backend.model.Meal;
+import com.library.backend.repository.MealRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+@Service
+
+public class MealService {
+
+    @Autowired
+    private MealRepository mealRepository;
```

Comment:
Instead of:
@Autowired
private MealRepository mealRepository;

create the constructor yourself:

private MealRepository mealRepository;

public MealService (MealRepository mealRepository) {
this mealRepository = mealRepository;
}

to avoid problems with injections as per SonarCloud:
Field dependency injection should be avoided [java:S6813](https://sonarcloud.io/organizations/sb0208/rules?open=java%3AS6813&rule_key=java%3AS6813)